README.md

```diff
@@ -1,2 +1,179 @@
-# metric-webapp
-React capstone project
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./banner.webp" alt="logo" />
+  <br/>
+
+  <h1><b>Metric-WebApp</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Metric-WEBAPP\] ](#-metric-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
```

Comment:
- [ ] As requested by the previous Reviewer and by the project requirement you should record a video presenting the features of the project. Kindly record a video of maximum length of 5 minutes to meet the requirements
1. The video  should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
2. The video should contain a demo of the project, highlighting all the project features.
3. The video should contain a highlight about a piece of the code you find interesting